src/components/forms/ClusterSpecificInput.tsx

```diff
@@ -16,6 +17,8 @@ interface Props {
   clusterMemberLinkTarget?: (member: string) => string;
   disabled?: boolean;
   helpText?: string;
+  placeholder?: string;
+  classname?: string;
```

Comment:
Naming is a bit imprecise, because this only applies to the input in the non-specific branch. Having it named `classname ` implies this is about the whole component and in all cases. Then again a more specific name might be a bit convoluted, so renaming might be not a good option either. Why do we need the additional classname on that specific input in the first place?